File: Components/RepositoryCategory/RepositoryCategoryInfo.vb
Imports System
Imports System.Configuration
Imports System.Data
Imports System.Data.SqlClient
Imports System.Data.SqlTypes
Imports System.Reflection
Imports System.Xml
Imports DotNetNuke
Imports DotNetNuke.Common.Utilities
Imports DotNetNuke.Common.Globals
Imports DotNetNuke.Services.Search

Namespace DotNetNuke.Modules.Repository
    Public Class RepositoryCategoryInfo

        Private _ItemID As Integer
        Private _ModuleId As Integer
        Private _Category As String
        Private _Parent As Integer
        Private _ViewOrder As Integer
        Private _Count As Integer

        ' initialization
        Public Sub New()
        End Sub

        ' public properties
        Public Property ItemId() As Integer
            Get
                Return _ItemID
            End Get
            Set(ByVal Value As Integer)
                _ItemID = Value
            End Set
        End Property

        ' public properties
        Public Property ModuleId() As Integer
            Get
                Return _ModuleId
            End Get
            Set(ByVal Value As Integer)
                _ModuleId = Value
            End Set
        End Property

        Public Property Category() As String
            Get
                Return _Category
            End Get
            Set(ByVal Value As String)
                _Category = Value
            End Set
        End Property

        Public Property Parent() As Integer
            Get
                Return _Parent
            End Get
            Set(ByVal Value As Integer)
                _Parent = Value
            End Set
        End Property

        Public Property ViewOrder() As Integer
            Get
                Return _ViewOrder
            End Get
            Set(ByVal Value As Integer)
                _ViewOrder = Value
            End Set
        End Property

        Public Property Count() As Integer
            Get
                Return _Count
            End Get
            Set(ByVal Value As Integer)
                _Count = Value
            End Set
        End Property

    End Class
End Namespace

File: Components/RepositoryCategory/RepositoryCategoryInfo.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Linq;
using System.Configuration;
using System.Data.SqlClient;
using System.Data.SqlTypes;
using System.Reflection;
using System.Xml;
using DotNetNuke;
using DotNetNuke.Common.Utilities;
using DotNetNuke.Common;
using DotNetNuke.Services.Search;

namespace DotNetNuke.Modules.Repository
{
	public class RepositoryCategoryInfo
	{

		private int _ItemID;
		private int _ModuleId;
		private string _Category;
		private int _Parent;
		private int _ViewOrder;

		private int _Count;
		// initialization
		public RepositoryCategoryInfo()
		{
		}

		// public properties
		public int ItemId {
			get { return _ItemID; }
			set { _ItemID = value; }
		}

		// public properties
		public int ModuleId {
			get { return _ModuleId; }
			set { _ModuleId = value; }
		}

		public string Category {
			get { return _Category; }
			set { _Category = value; }
		}

		public int Parent {
			get { return _Parent; }
			set { _Parent = value; }
		}

		public int ViewOrder {
			get { return _ViewOrder; }
			set { _ViewOrder = value; }
		}

		public int Count {
			get { return _Count; }
			set { _Count = value; }
		}

	}
}
